pyproject.toml

```
@@ -267,11 +261,8 @@ doc.help = "Live documentation server"
 doc.cmd = "mkdocs serve"
 
 ci.help = "Run all tasks in CI"
-ci.sequence = [
-    {cmd="pre-commit run --all-files"},
-    "cover",
-]
-ci.env = {SKIP = "no-commit-to-branch"}
+ci.sequence = [{ cmd = "pre-commit run --all-files" }, "cover"]
```

Same, it's harder to read the sequence and will produce harder to read diff in case of change:
```suggestion
ci.sequence = [
    { cmd = "pre-commit run --all-files" }, 
    "cover",
]
```

---

right... `,` is better